integrations/notion/integration.definition.ts

```diff
@@ -0,0 +1,111 @@
+import { IntegrationDefinition } from '@botpress/sdk'
+import { z } from 'zod'
+import { name } from './package.json'
+
+const emptyObject = z.object({})
+const anyObject = z.object({}).passthrough()
+
+export default new IntegrationDefinition({
+  name,
+  version: '0.0.1',
+  icon: 'icon.svg',
+  readme: 'readme.md',
+  configuration: {
+    schema: z.object({
+      /**
+       * The auth token for the integration [Notion Integrations](https://developers.notion.com/docs/authorization#internal-integration-auth-flow-set-up)
+       */
+      authToken: z.string().min(1),
+    }),
+  },
+  channels: {
+    comments: {
```

Comment:
**chore:** I don't think we should declare a channel for messaging if the integration cannot answer back yet